### ISSUE TITLE
Example fracs don't change the plots

### ISSUE THREAD

**Issue by shimwell** [aidancrilly/NeSST#20]
In the example we have two lines at the top

```python
nst.frac_D_default = 0.5
nst.frac_T_default = 0.5
```

changing these values does not change the plot.

perhaps we should pass in the values as args to the methods where they are needed

**Comment by shimwell** [comment]
for example we could do this instead

```
Y_DD = nst.yield_from_dt_yield_ratio('dd',Y_DT,Tion, 0.5, 0.5)
Y_TT = nst.yield_from_dt_yield_ratio('tt',Y_DT,Tion, 0.5, 0.5)
```

**Comment by aidancrilly** [comment]
fixed in PR #19
